great_expectations/validator/validator.py

```diff
@@ -606,8 +609,23 @@ def _validate_profiler_and_update_rules_properties(
                 metric_value_kwargs=metric_value_kwargs_override,
             )
 
-    @staticmethod
+        expectation_configuraiton_builders: List[ExpectationConfigurationBuilder] = (
+            rule.expectation_configuration_builders or []
+        )
+
+        expectation_configuraiton_builder: ExpectationConfigurationBuilder
+        for expectation_configuraiton_builder in expectation_configuraiton_builders:
```

Comment:
Nitpick - typo in configuration

Reply:
@cdkini Thank you for the great catch!  Very sorry -- fixed/committed.